Rakefile

```diff
@@ -70,3 +70,41 @@ task :populate, [:storage_root] => [:environment] do |_t, args|
     puts "You need to enter a specific storage root"
   end
 end
+
+desc "Fire off M2C existence check on a single storage root"
+task :m2c_exist_single_root, [:storage_root, :boolean, :profile] => [:environment] do |_t, args|
+  unless args[:profile] == 'profile' || args[:profile].nil?
+    p "Usage: rake m2c_exist_single_root[fixture_sr1,false] || rake m2c_exist_single_root[fixture_sr1, false, profile]"
```

Comment:
note here and in README:  use of spaces after commas inconsistent.  I prefer spaces, but consistency is much more important.